### ISSUE TITLE
Can we remove the facebook::react namespace for the module?

### ISSUE THREAD

**Issue by tomduncalf** [ericlewis/react-native-hostobject-demo#6]
Currently the module and its binding are in the `facebook::react` namespace... I think this could be a bit confusing as it implies that this is necessary, or that the code is part of React, whereas really you can use any namespace you like. Might it be worth changing it to something else to demonstrate this clearly?

**Comment by ericlewis** [comment]
Yeah, these don’t have to be namespaced. Feel free to open a PR if you like :)

**Comment by tomduncalf** [comment]
https://github.com/ericlewis/react-native-hostobject-demo/pull/7

**Comment by axemclion** [comment]
Yeah - native modules don't really need facebook::react namespace
